apps/services/sessions/src/app/sessions/sessions.service.ts

```diff
@@ -71,9 +72,15 @@ export class SessionsService {
     const device =
       browser || os ? `${browser}${browser && os ? ` (${os})` : os}` : undefined
 
+    const geoLocation = geoip.lookup(session.ip)
+    const ipLocation = geoLocation
+      ? `${geoLocation.city}, ${geoLocation.country}`
+      : undefined
```

Comment:
Does this need to be more defensive or are both properties always returned?

Reply:
Looking at the source the country could be returned with empty city. So I will make this more defensive on the string formatting.